pkgs/tools/networking/libreswan/default.nix

```
@@ -52,6 +53,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-9kLctjXpCVZMqP2Z6kSrQ/YHI7TXbBWO2BKXjEWzmLk=";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://libreswan.org/security/CVE-2023-23009/CVE-2023-23009-libreswan-4.4-4.9.patch";
```

I'm not sure if this is actually needed in this case, but I would do
```suggestion
      url = "https://libreswan.org/security/CVE-2023-23009/CVE-2023-23009-libreswan-4.4-4.9.patch";
      name = "CVE-2023-23009.patch";
```
so that [vulnix](https://github.com/flyingcircusio/vulnix#cve-patch-auto-detection) is sure to detect the software has been patched.

---

Does it not just look for substrings? Would have thought it would detect `CVE-2023-23009-libreswan-4.4-4.9.patch`

---

Maybe, I'm not sure, though.

---

It does, `CVE-2023-23009-libreswan-4.4-4.9.patch` is fine.

https://github.com/flyingcircusio/vulnix/blob/f56f3ac857626171b95e51d98cb6874278f789d3/src/vulnix/derivation.py#L122-L127